simple_utils.py

```
@@ -0,0 +1,13 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text: str) -> str:
+    """Reverses the characters in a string."""
+    return text[::-1]
+
+def count_words(sentence: str) -> int:
+    """Counts the number of words in a sentence."""
+    return len(sentence.split())
+
+def celsius_to_fahrenheit(celsius: float) -> float:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)